src/app/components/card/card.component.html

```diff
@@ -22,9 +22,20 @@
         <div class="highlights" *ngIf="card?.tabs.length === 0">
             <div class="section_title">{{ 'No Highlights' | translate}}</div>
         </div>
-        <img src="{{card?.preview}}" class="image" />
+        <ng-container *ngIf="card?.preview$ | async; let preview; else notPreview">
+            <img src="{{preview}}" class="image" />
+        </ng-container>
+        <ng-template #notPreview>
+            <img src="assets/no_preview.png" class="image" />
+        </ng-template>
     </div>
     <div class="groups">
+        <div *ngIf="card.organisation !=='no_organisation'">
+            <div class="section_title">{{ 'Organisation' | translate}}</div>
+            <mat-chip-list>
+                <mat-chip class="organisation" >{{card.organisation}}</mat-chip>
+            </mat-chip-list>
+        </div>
```

Comment:
Why adding organisation here, if they are in a section dedicated to the organisation ?

Reply:
fixed